gpii/node_modules/settingsHandlers/src/WebSocketsComponent.js

```diff
@@ -0,0 +1,198 @@
+/*!
+  GPII Web Sockets Settings Handler
+
+  Copyright 2014, 2015 Emergya
+
+  Licensed under the New BSD license. You may not use this file except in
+  compliance with this License.
+
+  You may obtain a copy of the License at
+  https://github.com/gpii/universal/LICENSE.txt
+*/
+
+/*global require*/
+
+(function () {
+
+    "use strict";
+
+    var fluid = require("infusion"),
+        gpii = fluid.registerNamespace("gpii"),
+        $ = fluid.registerNamespace("jQuery");
+
+    fluid.defaults("gpii.settingsHandlers.WebSocketsComponent", {
+        gradeNames: ["fluid.modelRelayComponent", "autoInit"],
+        changeApplierOptions: {
+            resolverGetConfig: fluid.model.escapedGetConfig,
+            resolverSetConfig: fluid.model.escapedSetConfig
+        },
+        clients: {},
```

Comment:
This should go into 
members: {
    clients: {}
}
and then be addressed directly at top level as that.clients - since a component's options are not meant to be mutable after construction